### PR TITLE
gluon-web-cellular: add German translation

### PR DESCRIPTION
**Pull request by herbetom**
image

### CODE REVIEW

**Review by blocktrron — APPROVED** [review]
lgtm

File: package/gluon-web-cellular/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+msgid "Username"
+msgstr "Nutzername"
+
+msgid ""
+"You can enable uplink via cellular service. If you decide so, the VPN "
+"connection is established using the integrated WWAN modem."
+msgstr ""
+"Du kannst den Uplink über einen Mobilfunk Service aktivieren. Wenn du "
+"dich dafür entscheidest wird die VPN Verbindung über das integrierte "
```
Comment (AiyionPrime) [inline package/gluon-web-cellular/i18n/de.po:48]:
```suggestion
"dich dafür entscheidest, wird die VPN-Verbindung über das integrierte "
```

File: package/gluon-web-cellular/i18n/de.po [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+"Mobilfunk Modem hergestellt."
```
Comment (AiyionPrime) [inline package/gluon-web-cellular/i18n/de.po:49]:
```suggestion
"Mobilfunkmodem hergestellt."
```
or
```suggestion
"Mobilfunk-Modem hergestellt."
```

would be fine, but the space is not.

**Review by AiyionPrime — CHANGES_REQUESTED** [review]
Other than those two it looks fine.

**Review by AiyionPrime — APPROVED** [review]

**Pull request merged**